pyext/src/rmsd_calculation.py

```diff
@@ -425,7 +426,12 @@ def get_rmsds_matrix(conforms,  mode,  sup,  cores, symm_groups=None):
     if mode == "cpu_omp":
         calculator.setNumberOfOpenMPThreads(int(cores))
 
-    rmsd = calculator.pairwiseRMSDMatrix()
+    if not symm_groups:
+        rmsd = calculator.pairwiseRMSDMatrix()
+    else:
+        rmsd = []
+        for i in range(len(conforms) - 1):
+            rmsd += list(calculator.oneVsFollowing(i))
```

Comment:
The main fix for ambiguity.

Reply:
This fixes Jan's issue on pyRMSD not considering ambiguity while calculating all vs all distances in pairwiseRMSDMatrix. This fix is simple and need not be parallelized further since calculator.oneVsFollowing is already parallelized.